Quorum/checks/new_listing.py

```diff
@@ -18,6 +17,10 @@ class FunctionCallDetails(BaseModel):
 
 
 class NewListingCheck(Check):
+    def __init__(self, customer, chain, proposal_address, source_codes):
+        super().__init__(customer, chain, proposal_address, source_codes)
+        self.check_path = "new_listings.json"
```

Comment:
This should either be a class variable or a global variable.

Reply:
Done